cci/CMakeLists.txt

```diff
@@ -196,7 +196,7 @@ if(WIN32)
   else(FOR_OTHER_DRIVER)
     target_compile_definitions(cascci PRIVATE CAS_CCI_DL)
   endif(FOR_OTHER_DRIVER)
-  set_target_properties(cascci PROPERTIES LINK_FLAGS "/DEF:\"${CMAKE_SOURCE_DIR}/win/cascci/cascci.def\"" LINK_FLAGS_RELEASE "/NODEFAULTLIB:libcmt.lib" LINK_FLAGS_DEBUG "/NODEFAULTLIB:msvcrt.lib")
+  set_target_properties(cascci PROPERTIES LINK_FLAGS "/DEF:\"${CCI_ROOT_PATH}/win/cascci/cascci.def\"" LINK_FLAGS_RELEASE "/NODEFAULTLIB:libcmt.lib" LINK_FLAGS_DEBUG "/NODEFAULTLIB:msvcrt.lib")
```

Comment:
If CUBRID/win/cascci/cascci.def is no longer used, shouldn't it be deleted from the CUBRID repository?

Reply:
It will be deleted like remarks comment.